src/controllers/Favorites.js

```diff
@@ -0,0 +1,133 @@
+/**
+ * @fileoverview Favorites Controller
+ *
+ * @author Ayelegun Kayode Michael
+ *
+ */
+import models from '../models';
+
+/**
+ * Controls endpoints for teams
+ * @class Favorites
+ */
+export default class Favorites {
+  /**
+    * @method addFavorite
+    * @desc This method adds a team to a user's favorites
+    *
+    * @param { object } request The request object
+    * @param { object } response The response object
+    *
+    * @returns { object } response
+    */
+  async addFavorite(request, response) {
+    try {
+      const { id } = request.user;
+      const existingTeam = await models.Team.findOne({
+        where: {
+          id: request.params.teamId,
+        }
+      });
+      if (!existingTeam) {
+        throw new Error('Team not found.');
+      }
+      const allFavorites = await models.Favorites.findAll({
+        where: {
+          teamId: existingTeam.dataValues.id
+        }
+      });
+      if (allFavorites.length === 0) {
+        const favoriteTeams = await models.Favorites.create({
+          userId: id,
+          teamId: existingTeam.dataValues.id,
+        });
+        return response.status(201).json({
```

Comment:
There's actually a way to send responses using `response.sendSuccess` or `response.sendFailure`